example-document/03-troubleshooting.md

```diff
@@ -0,0 +1,83 @@
+# Troubleshooting {#troubleshooting}
+
+Many pipeline failures are caused by justa few issues. This section describes the debugging process of a failed pipeline.
+
+First of all, to see the pipeline statu:
+1. Go to your repository
+2. Open Actions tab
+3. Open detail of your pipeline run
+4. Find out what job is failing
+5. See detailed logs of failed job
+6. Seach for commonly known error messages using native GitHub log search on top right corner
+
+## Validate YAML documents
+
+YAML formatting is strict, and all errors in this job are related to it. Search for `yaml.scanner.ScannerError` to see what went wrong. Example of the log:
+
+```
+yaml.scanner.ScannerError: mapping values are not allowed in this context
+  in "/__w/istqb-ctal-ta/istqb-ctal-ta/sample-exam-answers/questions.yml", line 562, column 19
+```
+To resolve these issues:
+1. Navigate to affected file (`sample-exam-answers/questions.yml` line 562, position 19 in this case)
+2. Resolve the formatting issue which could be cause by several things. The most common are:
+  * wrong indents - fix by adding 2 spaces per indent level
+  * using colon `:` or hashtag `#` in the text - fix by adding the text in quotes `"` or `'`
+  * missing pipe `|` in right after the key name to allow multiline inputs - fix by adding pipe `|` before multiline markdown input
```

Comment:
This section seems out-of-date after https://github.com/istqborg/istqb_product_base/pull/87.

Reply:
Section rewritten.